docs/docs/reference/policies.mdx

```diff
@@ -32,6 +32,28 @@ spec:
   path: my-script.rego
 ```
 
+### Supporting multiple material types
```

Comment:
I'd review the example above to only use this new spec format, not the previous one, which is now deprecated correct? We should also rewrite the policies we have in place to that no?